shared/Views/Shared/Components/CourseDetails/Default.cshtml

```diff
@@ -25,9 +25,9 @@
             <h1 class="govuk-heading-xl" role="heading">
                 @Model.Course.Name at @Model.Provider.Name
             
-                @if (!String.IsNullOrEmpty(Model.Course.AccreditingProvider.Name))
+                @if (!String.IsNullOrEmpty(Model.Course.AccreditingProvider?.Name))
                 {
-                    <span class="govuk-caption-xl">Accredited by @Model.Course.AccreditingProvider.Name</span>
+                    <span class="govuk-caption-xl">Accredited by @Model.Course.AccreditingProvider?.Name</span>
```

Comment:
Does this need a test?

Reply:
ASP.NET views are pretty untestable :(

Reply:
I shall refrain from questioning `untestable` statement....